pkg/proxy/winkernel/hns.go

```diff
@@ -147,26 +147,21 @@ func (hns hns) getAllEndpointsByNetwork(networkName string) (map[string]*(endpoi
 			serving:     true,
 			terminating: false,
 		}
-		endpointInfos[ep.IpConfigurations[0].IpAddress] = endpointInfos[ep.Id]
 
-		if len(ep.IpConfigurations) == 1 {
-			continue
-		}
+		endpointInfos[ep.Id] = epInfo
 
-		// If ipFamilyPolicy is RequireDualStack or PreferDualStack, then there will be 2 IPS (iPV4 and IPV6)
-		// in the endpoint list
-		endpointDualstack := &endpointInfo{
```

Comment:
Any reason we no longer need to store dual stack IP address in the endpoint info?

Reply:
We need dualstack information too. Instead of specifying the index, will iterate through the ep.IpConfigurations array to access dualstack info. In previous commit, I missed to keep epInfo construction part of the loop. I have added it now.